implementations/elixir/ockam/ockam/lib/ockam/identity/sidecar.ex

```diff
@@ -39,14 +46,14 @@ defmodule Ockam.Identity.Sidecar do
     end
   end
 
-  @spec create_signature(identity :: t(), auth_hash :: binary()) ::
+  @spec create_signature(vault_name :: String.t() | nil, identity :: t(), auth_hash :: binary()) ::
           {:ok, proof :: proof()} | {:error, reason :: any()}
-  def create_signature(identity, auth_hash) do
+  def create_signature(vault_name, identity, auth_hash) do
```

Comment:
Should we use default argument here, if we support `nil` as an argument?

Reply:
yes will do it

Reply:
Did it on the Identity module   that act as the api in front of sidecar/stub implementations